pkg/allocator/allocator.go

```diff
@@ -265,11 +265,15 @@ type Backend interface {
 // After creation, IDs can be allocated with Allocate() and released with
 // Release()
 func NewAllocator(typ AllocatorKey, backend Backend, opts ...AllocatorOption) (*Allocator, error) {
+	// Calculate initial ID range with cluster ID
+	minID := option.Config.ClusterID<<16 + 1
+	maxID := minID + (1<<16 - 1)
```

Comment:
Aren't these constants defined somewhere and we can import them?

Reply:
Hi @christarazi, thanks for your comments!

I found the `ClusterIDShift` defined in [pkg/identity](https://github.com/cilium/cilium/blob/master/pkg/identity/numericidentity.go#L19), but haven't found any place that defined the min/max range. Do I miss something?

Reply:
No I thought they existed, but seems not. This is fine, thanks